agents/che-core-api-agent/src/main/java/org/eclipse/che/api/agent/server/launcher/AbstractAgentLauncher.java

```diff
@@ -97,12 +97,20 @@ public void writeLine(String line) throws IOException {
           Thread.sleep(agentPingDelayMs);
         }
       }
+
```

Comment:
maybe we can delete this line of this PR

Reply:
done